README.md

```diff
@@ -1,7 +1,25 @@
-### Trace Visualization tutorial
+## Trace Visualization Labs
 
-This repository contains a series of labs and guides for collecting traces, from the operating system or userspace applications or cloud environments, and visualizing them with trace visualization tools.
+This repository contains a series of labs and guides for collecting traces from the operating system or userspace applications or cloud environments, and visualizing them with trace visualization tools. While most of the labs will use [LTTng](http://lttng.org) for trace collection and [Trace Compass](http://tracecompass.org/) for visualization, some labs may use other tools.
 
-While most of the labs will use [LTTng](http://lttng.org) for trace collection and [Trace Compass](http://tracecompass.org/) for visualization, some labs may use other tools.
+Each lab has specific goals and objectives. The labs are composed of various tasks to achieve the goal and in the process. Each lab has a README, which contains suggestions for navigation of the tasks. 
```

Comment:
Actually, the README _is_ the lab now, with the tasks in it. The first task or 2 involve reproducing the experiment, getting the trace and importing in Trace Compass, so can be skipped during the lisa talk if people use the traces from the sample trace archive.